lib/exception_handling/exception_catalog.rb

```diff
@@ -22,7 +22,7 @@ def refresh_filters
       end
 
     rescue => ex # any exceptions
-      ExceptionHandling::log_error(ex, "ExceptionRegexes::refresh_filters: #{@filter_path}", nil, true)
+      ExceptionHandling.write_exception_to_log(ex, "ExceptionRegexes::refresh_filters: #{@filter_path}", Time.now.to_i)
```

Comment:
Should probably add a comment explaining why you are using the non-standard logging